go/vt/vttablet/tabletserver/vstreamer/helper_event_test.go

```diff
@@ -511,8 +511,12 @@ func (ts *TestSpec) getFieldEvent(table *schemadiff.CreateTableEntity) *TestFiel
 			tc.len = int64(len(col.Type.EnumValues) + 1)
 			tc.colType = fmt.Sprintf("%s(%s)", tc.dataTypeLowered, strings.Join(col.Type.EnumValues, ","))
 			ts.metadata[getMetadataKey(table.Name(), tc.name)] = col.Type.EnumValues
+		case "json":
+			tc.colType = "json"
+			tc.len = lengthJSON
+			tc.collationID = collations.CollationBinaryID
 		default:
-			log.Infof(fmt.Sprintf("unknown sqlTypeString %s", tc.dataTypeLowered))
+			ts.t.Fatalf("unknown sqlTypeString %s", tc.dataTypeLowered)
```

Comment:
We should not be using `t.Fatal` in new code but using `require.Fail[Now]` instead. We've had efforts to remove/replace this usage across the code base in the past so we shouldn't introduce new usage -- that's my thinking.